### DISCUSSION TITLE
Leverage HTTPS Includes to manage scripts from a central location

### DISCUSSION THREAD

**Discussion by feathers1664** [ConSol-Monitoring/snclient#176] [Ideas]
I'm already using using the https include option to have a central managed configuration file for all my clients.

Is it possible to leverage this to populate the scripts folder from a central managed location as well?

**Comment by sni** [comment]
There are no plans to implement a general solution right now. But nobody prevents you from adding a "check" plugin/script which does exactly that and trigger that script regularly.
